core-field-editors/README.md

```diff
@@ -0,0 +1,32 @@
+Contentful Core Field Editors
+=============================
+
+This folder contains the source code to some of the Contentful Web App’s builtin
+field editors. The code targets the [Contentful UI Extension
+API][ui-extension-sdk]. This allows you to modify the source code, upload the
+editors as extensions to Contentful and use them in our App.
+
+To do this clone the repository and install the field editor dependencies.
+~~~
+git clone https://github.com/contentful/extensions.git contentful-extensions
+cd contentful-extensions/core-field-editors
+npm install
+npm install -g contentful-extension-cli
+~~~
+
+Move to the field editor folder and bundle the source code.
```

Comment:
Proposal:
"Then go to the field editor folder and create a bundle called `editor.dist.html`, which will be used as the extension's source:"